eng/pipelines/extra-platforms/runtime-extra-platforms-ioslike.yml

```diff
@@ -7,81 +7,120 @@ parameters:
+  #
+  - template: /eng/pipelines/common/platform-matrix.yml
+    parameters:
+      jobTemplate: /eng/pipelines/common/global-build-job.yml
+      helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
+      buildConfig: Release
+      runtimeFlavor: mono
+      isExtraPlatformsBuild: ${{ parameters.isExtraPlatformsBuild }}
+      isiOSLikeOnlyBuild: ${{ parameters.isiOSLikeOnlyBuild }}
+      platforms:
+        - ios_arm64
+        - tvos_arm64
+      variables:
+        - ${{ if and(eq(variables['System.TeamProject'], 'public'), eq(variables['Build.Reason'], 'PullRequest')) }}:
+          - name: _HelixSource
+            value: pr/dotnet/runtime/$(Build.SourceBranch)
+        - ${{ if and(eq(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest')) }}:
+          - name: _HelixSource
+            value: ci/dotnet/runtime/$(Build.SourceBranch)
+        - name: timeoutPerTestInMinutes
+          value: 60
+        - name: timeoutPerTestCollectionInMinutes
+          value: 180
+      jobParameters:
+        testGroup: innerloop
+        nameSuffix: AllSubsets_Mono_RuntimeTests
+        buildArgs: -s mono+libs -c $(_BuildConfig)
+        timeoutInMinutes: 240
+        # extra steps, run tests
+        extraVariablesTemplates:
+          - template: /eng/pipelines/common/templates/runtimes/test-variables.yml
+            parameters:
+              testGroup: innerloop
+        extraStepsTemplate: /eng/pipelines/common/templates/runtimes/build-runtime-tests-and-send-to-helix.yml
+        extraStepsParameters:
+          creator: dotnet-bot
+          # FIXME: Currently, tracing/eventpipe tests are only enabled on iOS platforms. It should be expanded to include all runtime tests. Tracking issue: https://github.com/dotnet/runtime/issues/84254
+          testBuildArgs: tree tracing/eventpipe
+          testRunNamePrefixSuffix: Mono_$(_BuildConfig)
+
+- ${{ if eq(parameters.isMonoOnlyBuild, false) }}:
+  #
+  # Build the whole product using NativeAOT for iOS/tvOS and run runtime tests with iOS/tvOS devices
+  #
+  - template: /eng/pipelines/common/platform-matrix.yml
+    parameters:
+      jobTemplate: /eng/pipelines/common/global-build-job.yml
+      helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
+      buildConfig: Release
+      runtimeFlavor: coreclr
+      isExtraPlatformsBuild: ${{ parameters.isExtraPlatformsBuild }}
+      isiOSLikeOnlyBuild: ${{ parameters.isiOSLikeOnlyBuild }}
+      platforms:
+        - ios_arm64
+        - tvos_arm64
+      variables:
+        - name: timeoutPerTestInMinutes
+          value: 60
+        - name: timeoutPerTestCollectionInMinutes
+          value: 180
+      jobParameters:
+        testGroup: innerloop
+        nameSuffix: AllSubsets_NativeAOT_RuntimeTests
+        timeoutInMinutes: 240
+        buildArgs: --cross -s clr.alljits+clr.tools+clr.nativeaotruntime+clr.nativeaotlibs+libs -c $(_BuildConfig) -rc $(_BuildConfig)
```

Comment:
Question: 
In several places, when cross build is performed I noticed that both `-c $(_BuildConfig) -rc $(_BuildConfig)` are used in the build, what is the reason that both flags are passed? 
Isn't the global `-c $(_BuildConfig)` applied to both runtime and libraries configuration?

Reply:
Thanks, I wasn't sure if it global.